rfcs/0018-glides-trainsheet-events.md

```diff
@@ -140,69 +105,125 @@ In order to reduce event duplication, a Trip Key is used to identify both added
 - `tripKey` (Trip Key): which trip is being updated.
 - `comment` (string, optional): free text information about the trip.
 - `startLocation` (Location, optional): the new destination of the train.
 - `endLocation` (Location, optional): the new destination of the train.
 - `startTime` (Time, optional): if present, the new time that the train is expected to depart `startLocation` (or the existing `startLocation` of the trip).
 - `endTime` (Time, optional): if present, the new time that the train is expected to arrive at `endLocation` (or the existing `endLocation` of the trip).
-- `consist` (array of Delta Car, optional): the cars assigned to perform this trip. If a car is `"cleared"`, then no car is currently assigned to that position in the consist. If a car is `"unmodified"` then the car is the value from the most-recent [Trip updated](#Trip-updated) event. The front car is listed first.
-- `scheduledOperators`: (array of Operator, optional): if present, the operators who were scheduled to operate this trip.
-- `operators` (array of Delta Operator, optional): the list of operators assigned to perform this trip. If an operator is `"cleared"`, then no operator is assigned to the car in that position in the consist. If the operator is `"unmodified"` then the operator is the value from the schedule or the most-recent [Trip updated](#Trip-updated) event. The operator of the front car is listed first.
-- `dropped`: (Dropped Reason or `false`, optional): if `false`, the trip is not dropped (and restored if previously dropped). If a [Dropped Reason](#dropped-reason), the reason the trip was dropped.
+- `cars` (array of [Car](#Car), optional): array of length 1 or 2, containing the car numbers and operators for each car in the train assigned to the trip. If absent, there are no changes to any cars. If present, the length of the array is the length of the train. In a two car train, the front car is listed first.
+- `dropped` (boolean, optional): whether the trip has been dropped. If `false`, the trip is not dropped (and restored if previously dropped). Added trips can be dropped and restored just like scheduled trips.
+- `droppedReason` ([DroppedReason](#DroppedReason) | null, optional): The reason the trip was dropped. SHOULD only exist if the trip has been dropped. SHOULD be `null` if the trip was restored.
+- `scheduled` ([Scheduled](#Scheduled) | null): For trips in Glides' schedule, this represents the scheduled data for the trip. It will always be present and won't change between subsequent updates to the same trip. For added trips, this will always be null.
 
 *Notes*
 - setting a new location or time does not modify the [Trip Key](#trip-key) for a scheduled trip.
-- `consist` and `operators` MUST be the length of the train: length 1 for a 1-car consist, length 2 for a 2-car consist.
+- if a trip is dropped, fields that are then irrelevant (such as `startTime` or `cars`) SHOULD NOT be set.
```

Comment:
Then you MAY set those fields. (In practice, Glides would probably send separate updates because the inspector takes separate actions, one to undrop, and one to set the field.)

This should clarify whether those fields automatically revert to their previous values. There's another place where I specified that they don't (removing and readding a car), so they probably shouldn't here either.

Reply:
Sorry, I meant the second part of your response: this should be clear about what happens to values previous set when the trip is undropped.

Reply:
clarified in https://github.com/mbta/technology-docs/pull/18/commits/9c31c800c1ad2335176fd23b76a3cb5fa213e6c8